drivers/hmis/app/models/hmis/hud/concerns/shared.rb

```diff
@@ -21,35 +21,47 @@ def as_warehouse
       "GrdaWarehouse::Hud::#{self.class.name.demodulize}".constantize.find(id)
     end
 
+    def self.enrollment_related_hud_class_names
```

Comment:
I don't love this approach, but I figured this was a place where we already listed them out.

Reply:
I'm ok with this here.  Probably eventually they should move to `HudUtility*` since they vary by year.